src/tests/Dashboard/action/dictionaryAction.test.js

```diff
@@ -185,6 +187,21 @@ describe('Test suite for dictionary actions', () => {
     });
   });
 
+  it('should alert a user and cancel the request if not logged in', () => {
+    const notifyMock = jest.fn();
+    notify.show = notifyMock;
+    localStorage.clear();
+    moxios.wait(() => {
+      moxios.requests.mostRecent();
```

Comment:
Are we doing anything with this request?

Reply:
Nothing, it's redundant. Fixed,, thank you